app/classifier/drawing-tools/components/DetailsSubTaskForm/DetailsSubTaskForm.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StickyModalForm from 'modal-form/sticky';
+import { Provider } from 'react-redux';
+import styled, { ThemeProvider } from 'styled-components';
+import theme from 'styled-theming';
+import Translate from 'react-translate-component';
+import { pxToRem, zooTheme } from '../../../../theme';
+import ModalFocus from '../../../../components/modal-focus';
+import TaskTranslations from '../../../tasks/translations';
+
+const SEMI_MODAL_FORM_STYLE = {
+  pointerEvents: 'all'
+};
+
+const SEMI_MODAL_UNDERLAY_STYLE = {
+  backgroundColor: 'rgba(0, 0, 0, 0)',
+  pointerEvents: 'none'
+};
+
+class DetailsSubTaskForm extends React.Component {
+  constructor() {
+    super();
+    this.handleDetailsChange = this.handleDetailsChange.bind(this);
+    this.handleDetailsFormClose = this.handleDetailsFormClose.bind(this);
+  }
+
+  static contextTypes = {
+    store: PropTypes.object
+  }
+
+  componentDidUpdate() {
+    // What is this even doing? When this was in the root component, it was always being called for any update
+    // Only thing changing is the transform prop. 
+    // I've passed all of the parent's props to this because I don't know what this was reacting to
+    // In the modal-form code, this method is for repositioning the underlay's width and height
+    // But the opacity is 0, so we can't see it!
+    // The class for the StickyForm is updating the styles for the pointer and form position
+    // but the behavior seems the same if I comment out the following line:
+    if (this.detailsForm) this.detailsForm.reposition();
```

Comment:
@simensta might know the answer to this. My best guess is it keeps details on the screen after a task annotation has changed eg. after typing text into a text box.